plugins/module_utils/network/sonic/utils/utils.py

```diff
@@ -365,29 +365,39 @@ def get_device_interface_naming_mode(module):
     return intf_naming_mode
 
 
-STANDARD_ETH_REGEXP = r"[e|E]th\s*\d+/\d+"
+STANDARD_ETH_REGEXP = r"[e|E]th\s*\d+/\d+$"
+STANDARD_EXT_ETH_REGEXP = r"[e|E]th\s*\d+/\d+[/\.]\d+"
 NATIVE_ETH_REGEXP = r"[e|E]th*\d+$"
 NATIVE_MODE = "native"
 STANDARD_MODE = "standard"
+STANDARD_EXT_MODE = "standard_extended"
 
 
 def find_intf_naming_mode(intf_name):
     ret_intf_naming_mode = NATIVE_MODE
 
     if re.search(STANDARD_ETH_REGEXP, intf_name):
         ret_intf_naming_mode = STANDARD_MODE
+    elif re.search(STANDARD_EXT_ETH_REGEXP, intf_name):
+        ret_intf_naming_mode = STANDARD_EXT_MODE
 
     return ret_intf_naming_mode
 
 
 def validate_intf_naming_mode(intf_name, module):
     global intf_naming_mode
+    compatible_input_naming_modes = {
+        'native': [NATIVE_MODE],
+        'standard': [STANDARD_MODE],
+        'standard-ext': [STANDARD_MODE, STANDARD_EXT_MODE]
+    }
+
```

Comment:
The modified `STANDARD_ETH_REGEXP` does not match for breakout interfaces and the earlier expression can be used.

Since the interfaces name format for 'standard' and 'standard-extended' do not change for REST APIs, the individual regexes are not required.
```suggestion
STANDARD_ETH_REGEXP = r"[e|E]th\s*\d+/\d+"
NATIVE_ETH_REGEXP = r"[e|E]th*\d+$"
NATIVE_MODE = "native"
STANDARD_MODE = "standard"


def find_intf_naming_mode(intf_name):
    ret_intf_naming_mode = NATIVE_MODE

    if re.search(STANDARD_ETH_REGEXP, intf_name):
        ret_intf_naming_mode = STANDARD_MODE

    return ret_intf_naming_mode


def validate_intf_naming_mode(intf_name, module):
    global intf_naming_mode
    compatible_input_naming_modes = {
        'native': [NATIVE_MODE],
        'standard': [STANDARD_MODE],
        'standard-ext': [STANDARD_MODE]
    }

```